### PR TITLE
Fall 2024

### PR DESCRIPTION
**Pull request by dear-clouds**

### CODE REVIEW

File: custom_mappings.yaml [hunk 1]
```diff
@@ -191,6 +365,8 @@ entries:
         anilist-id: 104308
       - season: 2
         anilist-id: 124136
+      - season: 3
+        anilist-id: 154473
```
Comment (evie-lau) [inline custom_mappings.yaml:369]: This season 3 is under the wrong title. Move up one under Arifureta

Comment (dear-clouds) [reply custom_mappings.yaml]: Fixed

File: custom_mappings.yaml [hunk 2]
```diff
@@ -1,4 +1,174 @@
...
+        anilist-id: 171637
+  - title: "Tonbo!"
+    synonyms:
+      - "Ooi! Tonbo"
+      - "オーイ! とんぼ"
+    seasons:
+      - season: 1
+        anilist-id: 164440
+      - season: 2
+        anilist-id: 178729
```
Comment (evie-lau) [inline custom_mappings.yaml:101]: In Tonbo, I think second season should be listed as `season: 1` with `start: 14`.
TVDB only shows 1 season including all 26 episodes from the 2 AniList entries

Comment (dear-clouds) [reply custom_mappings.yaml:101]: Oh looks like they changed this 2 days ago 😭
Fixed.

Comment (evie-lau) [reply custom_mappings.yaml:101]: I think you missed changing the season from `2` to `1`

**Comment by evie-lau** [comment]
Closing and reopening to test the season validation script

**Comment by dear-clouds** [comment]
Looks like it's not successful, is there anything else I need to fix?

**Comment by evie-lau** [comment]
It should be good to go now. We're having issues setting up the script to run here

**Pull request merged**